### PR TITLE
Sscs 10054 referred by tcw

### PR DESCRIPTION
**Pull request by paul-pearson**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/SSCS-10054

### Change description ###

Referred by TCW task

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/resources/wa-task-cancellation-sscs-benefit.dmn [hunk 1]
```diff
@@ -256,6 +256,29 @@
           <text>"reviewUrgentHearingRequest"</text>
         </outputEntry>
       </rule>
+      <rule id="DecisionRule_0gfe5m1">
+        <inputEntry id="UnaryTests_1pbpfql">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_0bp5hhr">
+          <text>"voidCase", "appealWithdrawn", "appealDormant", "confirmLapsed", "struckOut", "issueFinalDecision", "decisionIssued"</text>
```
Comment (markbrooks725) [inline src/main/resources/wa-task-cancellation-sscs-benefit.dmn:264]: "struckOut" should be a completion event.

File: src/main/resources/wa-task-initiation-sscs-benefit.dmn [hunk 2]
```diff
@@ -433,6 +478,41 @@ else 0</text>
...
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_11bt77z">
+          <text>"preHearingWork"</text>
+        </inputEntry>
+        <outputEntry id="LiteralExpression_0wid6ky">
+          <text>"referredByTcwPreHearing"</text>
+        </outputEntry>
+        <outputEntry id="LiteralExpression_01iy7nj">
+          <text>"Referred By TCW Pre Hearing"</text>
```
Comment (markbrooks725) [inline src/main/resources/wa-task-initiation-sscs-benefit.dmn:504]: Task name just needs to be "Referred By TCW"

**Review by markbrooks725 — COMMENTED** [review]
Couple of things to check.

File: src/main/resources/wa-task-cancellation-sscs-benefit.dmn (hunk 1 above)
Comment (paul-pearson) [reply src/main/resources/wa-task-cancellation-sscs-benefit.dmn]: Thanks Mark, now fixed.

File: src/main/resources/wa-task-initiation-sscs-benefit.dmn (hunk 2 above)
Comment (paul-pearson) [reply src/main/resources/wa-task-initiation-sscs-benefit.dmn]: Thanks, corrected.

**Pull request merged**